hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/AddUpdateRolloutWindowLayout.java

```diff
@@ -147,22 +143,24 @@
 
     private TextArea targetFilterQuery;
 
+    private final NullValidator nullValidator = new NullValidator(null, false);
+
     /**
      * Create components and layout.
      */
     public void init() {
-
         setSizeUndefined();
         createRequiredComponents();
         buildLayout();
     }
 
-    public CommonDialogWindow getWindow() {
-
-        addUpdateRolloutWindow = SPUIComponentProvider.getWindow(i18n.get("caption.configure.rollout"), null,
-                SPUIDefinitions.CREATE_UPDATE_WINDOW, this, event -> onRolloutSave(), event -> onDiscard(),
-                uiProperties.getLinks().getDocumentation().getRolloutView());
-        return addUpdateRolloutWindow;
+    public CommonDialogWindow getWindow(final Long rolloutId) {
```

Comment:
I saw a couple of calls like `getWindow(null)`. This is a good indicator that this method is doing to much.
Is it feasable to provide a second one like:

`getWindow()` which do not call `populateData(...)` or call `getWindow(null)` internal.